components/entitlement/org.wso2.carbon.identity.entitlement.endpoint/src/main/java/org/wso2/carbon/identity/entitlement/endpoint/util/JSONResponseWriter.java

```diff
@@ -115,13 +122,95 @@ private static JsonObject abstractResultToJSONObject(AbstractResult result) thro
             jsonResult.add(EntitlementEndpointConstants.ASSOCIATED_ADVICE, advices);
         }
 
+        // If include in Response enabled, other attributes will be populated from here.
```

Comment:
Shall we explain comment "If include in Response enabled"?. It is difficult to understand